### PR TITLE
Make attestantio import names consistent

### PR DESCRIPTION
**Pull request by jtraglia**
## 📝 Summary

Rename imports for github.com/attestantio/* to be more consistent.

See the mev-boost PR (in the references section) for more details. This PR follows the same conventions.

## ⛱ Motivation and Context

This has been a small pet-peeve of mine for a while. Due to a lot of overlapping package names, we were required to rename some imports. This wasn't very consistent. This PR aims to make things consistent.

## 📚 References

* https://github.com/flashbots/mev-boost/pull/560
* https://github.com/flashbots/mev-boost-relay/pull/509

---

## ✅ I have run these commands

* [ ] `make lint` (see other PR)
* [ ] `make test-race` (see other PR)
* [x] `go mod tidy`
* [x] I have seen and agree to `CONTRIBUTING.md`

### CODE REVIEW

**Review by michaelneuder — APPROVED** [review]
LGTM

**Comment by metachris** [comment]
this is great, thanks!

File: datastore/redis.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ func NewRedisCache(prefix, redisURI, readonlyURI string) (*RedisCache, error) {
 
 		prefixGetHeaderResponse:    fmt.Sprintf("%s/%s:cache-gethead-response", redisPrefix, prefix),
 		prefixExecPayloadCapella:   fmt.Sprintf("%s/%s:cache-execpayload-capella", redisPrefix, prefix),
-		prefixPayloadContentsDeneb: fmt.Sprintf("%s/%s:cache-payloadcontenst-deneb", redisPrefix, prefix),
+		prefixPayloadContentsDeneb: fmt.Sprintf("%s/%s:cache-payloadcontenst-builderApiDeneb", redisPrefix, prefix),
```
[the remark this answers is not in the document]
Comment (jtraglia) [reply datastore/redis.go:126]: Fix is here:

* https://github.com/flashbots/mev-boost-relay/pull/517

[the remark this answers is not in the document]
Comment (metachris) [reply datastore/redis.go:126]: whoops, thanks!

**Pull request merged**